CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+## Welcome!
+
+We're so glad you're thinking about contributing to an 18F open source project! If you're unsure about anything, just ask — or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
+
+We want to ensure a welcoming environment for all of our projects. Our staff follow the [18F Code of Conduct](https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md) and all contributors should do the same.
+
+We encourage you to read this project's CONTRIBUTING policy (you are here), its [LICENSE](LICENSE.md), and its [README](README.md).
+
+If you have any questions or want to read more, check out the [18F Open Source Policy GitHub repository]( https://github.com/18f/open-source-policy), or just [shoot us an email](mailto:18f@gsa.gov).
+
+## Pull request guidelines
+
+Please follow our [Git Protocol][git] and [Code Review][review] guidelines.
+[Glen Sanford's thoughts on code reviews][thoughts] are also well worth reading.
+
+[git]: https://pages.18f.gov/development-guide/git-protocol/
```

Comment:
I don't want to get into a long discussion about this, but this page recommends using fast-forward merges (and thus maintaining a liniar history), which is the opposite of what the github "Merge" button does.